demo/kie_image_demo.py

```diff
@@ -0,0 +1,326 @@
+"""KIE Image Demo.
+
+Combines Text Detection + Text Recognition + KIE stages. Configs can be
+provided for each of the stages. The output is an image containing bounding
+boxes around text with KIE annotations and a json file containing these
+annotations.
+"""
+
+from argparse import ArgumentParser
+
+import mmcv
+import numpy as np
+import torch
+from mmcv.image.misc import tensor2imgs
+from mmdet.apis import init_detector
+
+from mmocr.apis.inference import model_inference
+from mmocr.datasets.pipelines.box_utils import sort_vertex8
+from mmocr.datasets.pipelines.crop import crop_img
+from mmocr.utils.check_argument import is_type_list
+from mmocr.utils.fileio import list_from_file
+
+
+def pad_text_indices(text_inds):
```

Comment:
It will be better to use `mmocr/datasets/kie_dataset.py` directly, instead of copying these functions here.